bears/general/AnnotationBear.py

```diff
@@ -58,7 +62,10 @@ def run(self, filename, file, language: str, coalang_dir: str = None):
             yield Result(self, str(e), severity=RESULT_SEVERITY.MAJOR,
                          affected_code=(e.code,))
 
-        content = {'strings': string_ranges, 'comments': comment_ranges}
+        content = {'singleline strings': ranges[0],
+                   'multiline strings': ranges[1],
+                   'singleline comments': ranges[2],
+                   'multiline comments': ranges[3]}
```

Comment:
I'd love to see some classes too, instead of returning this dict rater make it a list of:

```python
@attr.s
class String:
    start = attr.ib(convert=SourcePosition)
    content_start = attr.ib(convert=SourcePosition)
    content_end = attr.ib(convert=SourcePosition)
    end = attr.ib(convert=SourcePosition)
```

plus some convenience properties